src/components/Table/TableCell.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import React from 'react';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+import TableHeadContext from './TableHeadContext';
+
+
+const sharedStyle = css`
+    padding: 0 20px;
+    &:first-child {
+        padding-left: 0;
+    }
+
+    &:last-child {
+        padding-right: 0;
+    }
+
+    ${get('typography.table')};
+
+    ${COLOR};
+    ${COMMON};
+    ${TYPOGRAPHY};
+`;
+
+const TableCellWrapper = styled.td`
```

Comment:
In other areas of the code we usually append `Base` instead of `Wrapper` perhaps `TableCellBase` fits better 

You can also can do a similar change for `TableHeadCell`